invenio_records_rest/config.py

```diff
@@ -96,6 +96,7 @@ def deleted_pid_error_handler(error):
             'pid_fetcher': '<registered-pid-fetcher>',
             'pid_minter': '<registered-minter-name>',
             'pid_type': '<record-pid-type>',
+            'read_list_permission_factory_imp': permission_check_factory(),
```

Comment:
just seeing this now - is it not better named ``list_permission_factory``? create, read, update, delete, list

Reply:
Yeah maybe, if post_list is create, maybe this should be list. Should I do another PR? Sorry for the inconvenience :/

Reply:
@lnielsen yes you are right, for me it was clear enough, but it is indeed more consistent with the rest of the code. We can see how to proceed.